pkg/semtech-loramac/contrib/semtech_loramac_arch.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2017 Fundacion Inria Chile
+ *               2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     pkg_semtech-loramac
+ * @file
+ * @brief       Compatibility system functions used in LoRaMAC
+ *
+ * @author      Jose Ignacio Alamos <jialamos@uc.cl>
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include "semtech-loramac/board.h"
+#include "random.h"
+
+void srand1(uint32_t seed)
+{
+    random_init(seed);
```

Comment:
can this be avoided? If ```srand1()``` is only called once at init time, maybe an empty stub is ok, considering that random_init() gets called by auto init.

Reply:
You are right I think, I removed this call and still works.